packages/hideLiterals/src/index.ts

```diff
@@ -0,0 +1,35 @@
+// Replace numeric, string, list, and object literals with "empty" values.
```

Comment:
should allude to stripSensitiveLiterals in the same way that that function alludes to this one.

alternatively, just have one function with an option argument determining whether to strip lists and objects. (we are actually likely to consider changing the signature algorithm to not strip those soon anyway as part of a Pulsar project)

Reply:
I liked your alternative so went with that!